### DISCUSSION TITLE
Feasibility to Change Atom Order in dpdata

### DISCUSSION THREAD

**Discussion by Gemstrom** [deepmodeling/dpdata#766] [General]
Hi folks, 

I am currently using dpdata 0.2.16 for cpu and I read-in a deepmd raw data of a binary compound into a Labeled System. I wonder if there's any way I could reverse the order of elements? e.g. I have systems['atom_names']=['A', 'B'] and I want to get a LabeledSystem object with ['atom_names']=['B', 'A']. And I wonder if these two forms would be treated differently when using descriptor 'se_e2_a'.

Thank you so much!

**Comment by wanghan-iapcm** [comment]
The order of atom name and types does not matter in the deepmd-kit training.

**Reply by Gemstrom** [reply]
Thank you!
